### PR TITLE
Some gear-wasm-builder improvements

### PR DESCRIPTION
**Pull request by ark0f**
Copy WASM files to standard directory (`target/wasm32-unknown-unknown/{debug, release}`)

### CODE REVIEW

File: utils/wasm-builder/README.md [hunk 1]
```diff
@@ -47,17 +47,17 @@ mod code {
 }
 
 #[test]
-fn debug_wasm() {
+fn assert_wasm() {
     assert_eq!(
-        std::fs::read("target/wasm32-unknown-unknown/debug/test_program.wasm").unwrap(),
```
Comment (breathx) [inline utils/wasm-builder/README.md:52]: Am I right that you want to always push binaries to release dir, even if it had built without release flag?

File: utils/wasm-builder/src/cargo_command.rs [hunk 2]
```diff
@@ -26,7 +26,6 @@ pub struct CargoCommand {
     path: String,
     manifest_path: PathBuf,
     args: Vec<&'static str>,
-    profile: String,
```
Comment (breathx) [inline utils/wasm-builder/src/cargo_command.rs:29]: what about to really use profile setting?

**Comment by breathx** [comment]
I don't see any reason to force always build in release mode. If developer writes some functionality and regularly runs tests, there is no need to wait excess time

**Comment by ark0f** [comment]
> I don't see any reason to force always build in release mode. If developer writes some functionality and regularly runs tests, there is no need to wait excess time

But you even approved that logic: https://github.com/gear-tech/gear/pull/687

File: utils/wasm-builder/README.md (hunk 1 above)
Comment (shamilsan) [reply utils/wasm-builder/README.md:52]: Yes, we build WASMs in release mode but the binaries path should correspond to the profile and be intuitive for the user. Also, we had an idea to cut gstd's `debug` feature when building in release profile.

Comment (ark0f) [reply utils/wasm-builder/README.md]: > Also, we had an idea to cut gstd's debug feature when building in release profile.

I think we need to dicuss possibility and how to do it correctly

**Review by shamilsan — CHANGES_REQUESTED** [review]
Need to make `cargo` build the WASM with a selected profile instead of hard coded `--release`.

I guess it is better to make the debug-feature-related functionality (removing `debug` feature from `gstd` in release build) as a separate PR later.

**Review by breathx — APPROVED** [review]

**Review by shamilsan — APPROVED** [review]
Great 👍

**Pull request merged**